compiler/src/codegen/compcore.ml

```diff
@@ -1999,7 +1989,6 @@ let compile_exports env {functions; imports; exports; num_globals} =
           };
           add_dummy_loc {
             Ast.name=encode_string (Ident.name table_size);
-            (* We add one here because of heap top *)
```

Comment:
Oddly enough, changing this misaligns the global table, even with the removal of heap_adjust.

Reply:
I also tried just changing this to `compute_table_size` but that's not correct either.